ethdb/remote/remotedbserver/server.go

```diff
@@ -27,6 +28,58 @@ type KvServer struct {
 	kv ethdb.KV
 }
 
+func StartGrpcWithTLS(kv ethdb.KV, eth core.Backend, addr string, certFile string, keyFile string) {
```

Comment:
this method is a bit too much copy-paste from `StartGrpc` and it differs only in in a couple of places.
can you somehow generalize that?

I see that there are just 2 differences:
(1) log message
(2) credentials passing